src/ert/run_models/base_run_model.py

```diff
@@ -239,7 +239,7 @@ def queue_system(self) -> QueueSystem:
         return self._queue_config.queue_system
 
     @property
-    def _ensemble_size(self) -> int:
+    def ensemble_size(self) -> int:
```

Comment:
Is it intentional to make this public?

Reply:
Yes, it was already a competing public property for most of the run models, for example:
 
```python
self.ensemble_size = simulation_arguments.ensemble_size
```

Reply:
https://github.com/equinor/ert/pull/8391/commits/da52bd501cde5e281189421f4471d93b2e887055